query/iam/iam_subscription_owner_more_than_1.sql

```diff
@@ -0,0 +1,29 @@
+with owner_roles as (
+  select
+    role_name,
+    role_type, name,
+    title,
+    subscription_id
+  from
+    azure_role_definition
```

Comment:
we need to check from `azure_role_assignment` whether `Owner` azure_role_definition is assigned to users.
Please check this once.

Reply:
updated the script using the azure_role_definition and azure_role_assignment tables